ledger/src/snapshot_utils.rs

```diff
@@ -602,6 +613,12 @@ where
                 }
             };
             info!("Rebuilding accounts...");
+
+            // Convert bank to current version
+            let mut bank = match versioned_bank {
+                BankVersions::Bank1_0(bank_1_0) => bank_1_0.convert_to_current(),
+                BankVersions::Current(current_bank) => current_bank,
+            };
```

Comment:
I think you can ditch `BankVersions` entirely with:
```suggestion
            let mut bank = match snapshot_version {
                SNAPSHOT_VERSION_1_0 => ...
                SNAPSHOT_VERSION => ...
            };
```

Reply:
yup definitely unnecessary, removed